cssom-view-1/Overview.bs

```diff
@@ -359,13 +359,53 @@ the following steps must be run:
   </ul>
   ...then perform a <a>smooth scroll</a> of <var>box</var> to <var>position</var>. Once the position has finished updating, emit the <a event>scrollend</a> event.
   Otherwise, perform an <a>instant scroll</a> of <var>box</var> to <var>position</var>. After an <a>instant scroll</a> emit the <a event>scrollend</a> event.
-  
-  Note: If the scroll position did not change as a result of the user interaction or programmatic invocation, where no translations were applied as a result, then no <a event>scrollend</a> event fires because no scrolling occured.
 
-  Note: In the case the VisualViewport was scrolled, yielding no translation changes in the document, emit the <a event>scrollend</a> on the VisualViewport just as the <a event>scroll</a> event is emitted for VisualViewport scrolling.
+  Note: If the scroll position did not change as a result of the user interaction or programmatic invocation, where no translations were applied as a result, then no <a event>scrollend</a> event fires because no scrolling occured.
 </ol>
 
-Scroll is <dfn lt="scroll completed">completed</dfn> when the scroll position has no more pending updates or translations and the user has completed their gesture. Scroll position updates include smooth or instant mouse wheel scrolling, keyboard scrolling, scroll-snap events, or other APIs and gestures which cause the scroll position to update and possibly interpolate. User gestures like touch panning or trackpad scrolling aren't complete until pointers or keys have released. 
+When a user agent is to <dfn for="viewport" export>perform a scroll</dfn> of a <a>viewport</a> to a given position <var>position</var> and optionally a scroll behavior <var>behavior</var>
+(which is "<code>auto</code>" if omitted) it must perform a coordinated viewport scroll by following these steps:
```

Comment:
As in: we're coordinating a scroll distributed between the visual viewport's `scrolling box` and the document's viewport's (i.e. "layout viewport") `scrolling box`.

Basically "perform a scroll" for a `scrolling box` follows the previously defined steps while "perform a scroll" for a `viewport` is "overloaded" to perform this "coordination" by splitting up the scroll and calling "perform a scroll" on each of the viewport's `scrolling boxes`.

If "coordinated" isn't adding anything (other than confusion) I'm happy to remove it. Or perhaps give these steps a separate name (e.g. "perform a viewport scroll")?

Reply:
@smfr ping

Reply:
@smfr PTAL

Reply:
Meta comment: I think it might be clearer to refer to a “pan” when moving the visual viewport, and a “scroll” when moving the layout viewport (even though this text talks about firing a “scroll” event on the visual viewport).

When the visual viewport moves without a corresponding layout viewport change, the scrollbar doesn’t change.

Reply:
(Just mentioning for completeness, Firefox's scrollbars also reflect the visual viewport, i.e. shrink when zooming in and move when the visual viewport is scrolled.)